presto-jdbc/src/main/java/com/facebook/presto/jdbc/ColumnInfo.java

```diff
@@ -227,6 +233,8 @@ private static int getType(TypeSignature type)
                 return Types.DATE;
             case "decimal":
                 return Types.DECIMAL;
+            case "float":
+                return Types.REAL;
```

Comment:
I'm fine with that if the default precision turns out to be a value that agrees with the precision of a REAL. I haven't looked at the spec closely enough to know what it says about that.

FWIW, in SQL server, FLOAT with no precision argument is the same as DOUBLE PRECISION.